switchbot_ros/scripts/switchbot_ros_server.py

```diff
@@ -26,9 +26,20 @@ def __init__(self):
                 self.token = f.read().replace('\n', '')
         else:
             self.token = token
+
+        # Switchbot API v1.1 needs secret key
+        secret = rospy.get_param('~secret')
```

Comment:
```suggestion
        secret = rospy.get_param('~secret', None)
```

Reply:
I added  `, None` as you suggested.
https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/491/commits/429cd88063ae3b2785f682388c4939607b507d73#diff-1690cbebb7bf92ad5c7be4524a54f4c0566916a4da9a6b0c493666e891db80c9R31